### DISCUSSION TITLE
Unable to update directly from pendrive

### DISCUSSION THREAD

**Discussion by AdityaICVS** [rauc/rauc#1422] [Q&A]
Hello,

I think I am missing something very basic here. RAUC says "No intermediate storage on target required" but I am unable to perform updates without copying the files on target first. I am getting **unsafe bundle permissions 0770** error when I try to perform update directly from the pendrive. 

After copying the file, the update goes successfully without changing the permissions.

Any idea what could be wrong here? Any help would be appreciated.

Thanks

**Comment by jluebbe** [comment]
You're using a bundle in plain format, which cannot protect against modifications during installation if non-root users are allowed to write (see https://rauc.readthedocs.io/en/latest/advanced.html#protection-against-concurrent-bundle-modification for details). Ideally, you'd just switch to the verity bundle format: https://rauc.readthedocs.io/en/latest/reference.html#sec-ref-formats
